DOCS-8916 Add Using LLM Observability and APM Guide (#25352)
DataDog/documentation: content/en/llm_observability/guide/llm_observability_and_apm.md

--- 
title: Using LLM Observability and APM
description: Learn how to navigate between LLM Observability spans and APM spans so you can gain insights into LLM-specific operations and your broader application ecosystem.
further_reading: 
- link: "/llm_observability/terms/" 
  tag: "Documentation" 
  text: "Learn about LLM Observability Spans"
- link: "/glossary/#span/" 
  tag: "Documentation" 
  text: "Learn about APM Spans"
---

## Overview

> The LLM Observability SDK is built on APM's `dd-tracer`. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]. By instrumenting your LLM-specific operations with LLM Observability and your broader application with APM, LLM Observability and APM [spans][6] are linked in Datadog.

Reviewer: ```suggestion
The LLM Observability SDK is built on APM's `dd-tracer`. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]. You can link LLM Observability and APM [spans][6] in Datadog by instrumenting your LLM-specific operations with LLM Observability and your broader application with APM.
```
The last sentence tripped me up a bit. Rephrasing also gets rid of the passive.

> The LLM Observability SDK is built on APM's `dd-tracer`. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]. You can link LLM Observability and APM [spans][6] in Datadog by instrumenting your LLM-specific operations with LLM Observability and your broader application with APM.

Reviewer 2: ```suggestion
This guide explains how you can use both LLM Observability and APM to link LLM Observability and APM [spans][6] in Datadog. 

By instrumenting your LLM-specific operations with LLM Observability and your broader application with APM, you can accomplish the following:

```

> This integration allows you to accomplish the following:

Reviewer 2: ```suggestion

```

* Understand end-to-end visibility: Explore upstream and downstream requests of your LLM applications within the context of your entire application.

> * Dive deeper from APM to LLM Observability to investigate whether or not an issue with your application is specific to LLM-specific applications, such as a call to OpenAI.

Reviewer: ```suggestion
* From APM, dive deeper into LLM Observability: Investigate whether or not an issue with your application is specific to LLM-specific applications, such as a call to OpenAI.
```

## Setup

> 

Reviewer 2: Add
> The LLM Observability SDK is built on APM's `dd-tracer`. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7].

> If you are using the [LLM Observability SDK for Python][1] along with APM's [`dd-tracer`][2], you can navigate between spans in Datadog APM and LLM Observability without any additional setup.

Reviewer: ```suggestion
If you are using the [LLM Observability SDK for Python][1] along with APM's [`dd-tracer`][2], you can navigate between spans in Datadog APM and LLM Observability without additional setup.
```

--- 
title: Using LLM Observability and APM
description: Learn how to navigate between LLM Observability spans and APM spans so you can gain insights into LLM-specific operations and your broader application ecosystem.
further_reading: 
- link: "/llm_observability/terms/" 
  tag: "Documentation" 
  text: "Learn about LLM Observability Spans"
- link: "/glossary/#span/" 
  tag: "Documentation" 
  text: "Learn about APM Spans"
---

## Overview

This guide explains how you can use both LLM Observability and APM to link LLM Observability and APM [spans][6] in Datadog. 

By instrumenting your LLM-specific operations with LLM Observability and your broader application with APM, you can accomplish the following:

* Understand end-to-end visibility: Explore upstream and downstream requests of your LLM applications within the context of your entire application.
* From APM, dive deeper into LLM Observability: Investigate whether or not an issue with your application is specific to LLM-specific applications, such as a call to OpenAI.

## Setup

> The LLM Observability SDK is built on APM's dd-tracer. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]

Reviewer 3: ```suggestion
The LLM Observability SDK is built on APM's dd-tracer. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]

```
